website/docs/docs/core/connect-data-platform/redshift-setup.md

```diff
@@ -168,14 +165,59 @@ please refer to the official AWS documentation on [Configuration and credential
 
 </File>
 
+#### Specifying an IAM Profile
+
+When the `iam_profile` configuration is set, dbt will use the specified profile from your `~/.aws/config` file instead of using the profile name `default`
+
 </TabItem>
 
-</Tabs>
+<TabItem value="iam-user-inline">
+
+The following table lists the authentication parameters to use IAM authentication **on dbt Cloud**.
```

Comment:
We don't generally have the Cloud instructions on the Core adapter pages. My concern is that a user will search, end up here, and maybe not realize the rest is Core related only or become confused. Would it be enough to only include these on the Cloud adapter setup page?

Reply:
Moved everything in the following commit, thanks Matt, good call :)